readme.md

```diff
@@ -1389,7 +1426,7 @@ Changes should go to upstream repos and [`languages.yml`][languages-yml] in
 ## Types
 
 This package is fully typed with [TypeScript][].
-It exports additional `Grammar` and `Root` types that model their respective
+It exports additional `Grammar`, `Root`, `Options` and `GetOnigurumaUrl` types that model their respective
 interfaces.
```

Comment:
```suggestion
It exports additional `Grammar`, `Root`, `Options`, and `GetOnigurumaUrl`
types that model their respective interfaces.
```

Oxford comma is used throughout all docs.
Lines are wrapped at 80.

Reply:
done with 63eb1bd